src/pages/Dashboard/Modals/SupplyWithdraw/SupplyWithdrawForm.tsx

```diff
@@ -0,0 +1,202 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState, useEffect } from 'react';
+import BigNumber from 'bignumber.js';
+import { FormikProps } from 'formik';
+import { Typography } from '@mui/material';
+import { FormValues } from 'containers/AmountForm/validationSchema';
+import { AmountForm, IAmountFormProps } from 'containers/AmountForm';
+import {
+  TokenTextField,
+  Delimiter,
+  LabeledInlineContent,
+  ILabeledInlineContentProps,
+  PrimaryButton,
+  ProgressBarHorizontal,
+  Tooltip,
+  ValueUpdate,
+} from 'components';
+import { useTranslation } from 'translation';
+import { Asset, TokenId } from 'types';
+import {
+  getBigNumber,
+  convertCoinsToWei,
+  convertWeiToCoins,
+  formatCentsToReadableValue,
+  format,
+  currencyFormatter,
+} from 'utilities/common';
+import { useStyles } from '../styles';
+
+interface ISupplyWithdrawFormProps {
+  asset: Asset;
+  tokenInfo: ILabeledInlineContentProps[];
+  userTotalBorrowBalance: BigNumber;
+  userTotalBorrowLimit: BigNumber;
+  dailyEarnings: BigNumber;
+  onSubmit: IAmountFormProps['onSubmit'];
+  enabledButtonKey: string;
+  disabledButtonKey: string;
+  maxInput: BigNumber;
+  calculateNewBalance: (amount: BigNumber) => BigNumber;
+}
+
+export const SupplyWithdrawContent: React.FC<
+  Omit<ISupplyWithdrawFormProps, 'onSubmit'> & FormikProps<FormValues>
+> = ({
```

Comment:
So realizing that the prop types at the top of the file are the container's types, I think we should take this case the other way around. The container wraps the UI, so technically it should be the container's type that extends the UI. What do you think?